website/components/HelpDropdown.tsx

```diff
@@ -84,8 +81,8 @@ export default function HelpDropdown(): ReactElement {
         footer={<Button onClick={closeVersionModal}>Close</Button>}
       >
         <FlexColumnAlignCenter $gap={0}>
-          <p style={{ margin: 0 }}>Website v{WEBSITE3DCELLVIEWER_VERSION}</p>
-          <p style={{ margin: 0 }}>Volume viewer plugin v{VOLUMEVIEWER_VERSION}</p>
+          <p style={{ margin: 0 }}>Vole-E App v{VOLEAPP_VERSION}</p>
+          <p style={{ margin: 0 }}>Vol-E Core plugin v{VOLECORE_VERSION}</p>
```

Comment:
Not your word, but while we're here: "plugin" seems like odd/old terminology. Package?